ecosystem/sep-0010.md

```diff
@@ -94,9 +94,24 @@ Every other HTTP status code will be considered an error. For example:
 
 ### Token
 
-This endpoint accepts a signed challenge transaction (that was previously returned by the [`challenge`](#challenge) endpoint). If the client's signature is valid, it responds with a [JSON Web Token](https://jwt.io/) authenticating the user.
+This endpoint accepts a signed challenge transaction, validates it and responds with a session [JSON Web Token](https://jwt.io/) authenticating the user.
 
-It is recommended to include the following claims in JWT:
+Client submits a challenge transaction (that was previously returned by the [`challenge`](#challenge) endpoint) as a HTTP POST request to `WEB_AUTH_ENDPOINT` using one of the following formats (both should be equally supported by the server):
```

Comment:
@nebolsin do you think it's worth it to ask servers to implement both? It seems like some implementation overhead not much obvious benefit to me.

Reply:
@nebolsin that's a strong argument for supporting both. I'm merging this in now.